test/e2e/docs/style-guide.md

```diff
@@ -25,21 +25,23 @@ We don't chain function calls together and avoid using `.then` calls.
 Avoid doing:
 
 ```
-async selectContinue() {
-	const continueSelector = By.css( '.card[data-e2e-type="continue"] button' );
-	return await driverHelper
-		.waitTillPresentAndDisplayed( this.driver, continueSelector )
-		.then( () => driverHelper.clickWhenClickable( this.driver, continueSelector ) );
-	}
+async function openModal() {
+	const modalButtonLocator = By.css( 'button.open-modal' );
+	const modalLocator = By.css( '.modal' );
+	await driverHelper.clickWhenClickable( this.driver, modalButtonLocator ).then( async function () {
+		await driverHelper.waitUntilLocatedAndVisible( modalLocator );
```

Comment:
IMO, the previous example might be a bit better because it's relying on promises much more -- I think people would be much less likely to implement async/await inside a promise callback like this just because part of the point of async/await is to avoid `.then`.) In other words the previous example is like "don't use promises directly", and this one is like "don't do async/await poorly". What do you think?

Reply:
Good point, I agree! 🙌  Will remove the async from the callback 👍

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/51506/commits/9273124232190539f4e775a9c06e9668472710b6